app/assets/stylesheets/bootstrap-variables.scss

```diff
@@ -1,3 +1,5 @@
+@import "colors";
```

Comment:
Maybe we should not import `colors` in application anymore, or maybe we should import it there before bootstrap?